sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/documentanalysis/DocumentAnalysisAsyncClientTest.java

```diff
@@ -1219,4 +1219,16 @@ public void analyzeW2Data(HttpClient httpClient, DocumentAnalysisServiceVersion
             validateW2Data(syncPoller.getFinalResult());
         }, W2_JPG);
     }
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.formrecognizer.documentanalysis.TestUtils#getTestParameters")
+    public void analyzeDocumentInvalidLength(HttpClient httpClient, DocumentAnalysisServiceVersion serviceVersion) {
```

Comment:
Is it possible to just add `@DoNotRecord` annotation here to skip recording and just run this client-side only test? If that works, then we don't need those empty json recordings.